pkgs/tools/security/otpgen/default.nix

```diff
@@ -0,0 +1,40 @@
+{ 
+  pkgs
+  , lib
+  , stdenv
+  , makeWrapper
+  , fetchFromGitHub
+  , cmake
+  , pkg-config
+  , cryptopp
+  , qtbase
+}:
+
+stdenv.mkDerivation rec {
+
+  pname = "OTPGen";
+  version = "0.9.3";
+
+  src = fetchFromGitHub {
+    owner = "magiruuvelvet";
+    repo = "OTPGen";
+    rev = "v${version}";
+    sha256 = "19nyr3gmfxak0zdzibs1pjcdaxyy8x7jvrhgi3bsg9jbgf1m6xsr";
+  };
+
+  patches = [ ./core-cmake-cxx.patch ./cmakelists.patch ];
+
+  nativeBuildInputs = [ cmake  ];
+
+  depsBuildBuild = [ pkg-config ];
```

Comment:
Why specifically `depsBuildBuild` instead of `depsBuildHost` (aka `nativeBuildInputs`)?

Reply:
Fixed here https://github.com/NixOS/nixpkgs/pull/87887/commits/664712ac31668923f8c6de57aaaac4f4379ed927.